joda-money/src/main/java/com/fasterxml/jackson/datatype/jodamoney/MoneyDeserializer.java

```diff
@@ -16,15 +16,19 @@
 import org.joda.money.CurrencyUnit;
 import org.joda.money.Money;
 
+import static java.util.Objects.requireNonNull;
+
 public class MoneyDeserializer extends StdDeserializer<Money>
 {
     private static final long serialVersionUID = 1L;
 
     private final String F_AMOUNT = "amount";
     private final String F_CURRENCY = "currency";
+    private final AmountConverter<?> amountConverter;
 
-    public MoneyDeserializer() {
+    public MoneyDeserializer(final AmountConverter<?> amountConverter) {
```

Comment:
For backwards-compatibility may want to add `@Deprecated` no-args constructor (to be removed from 3.0)

Reply:
thanks, good catch :+1: the class if public after all, I missed that somehow